services/core-web/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -72,11 +74,31 @@ export const renderHighlightedTextColumn = (
   };
 };
 
-const withTag = (text: string, elem: ReactNode) => {
+const withTag = (record: any, elem: ReactNode, title: string) => {
   return (
-    <div className="inline-flex flex-between">
+    <div
+      className="inline-flex flex-between file-name-container"
+      style={!record.number_of_versions ? { marginLeft: "0" } : { marginLeft: "14px" }}
```

Comment:
I think this function needs a rename if it's going to be very specifically for document tags and not reusable for other use cases.

Reply:
This has been renamed now to `documentWithTag`